src/references.md

```diff
@@ -17,161 +17,140 @@ Which obey the following rules:
 * A reference cannot outlive its referent
 * A mutable reference cannot be aliased
 
-That's it. That's the whole model. Of course, we should probably define
-what *aliased* means. To define aliasing, we must define the notion of
-*paths* and *liveness*.
+That's it. That's the whole model references follow.
 
+Of course, we should probably define what *aliased* means. An aliased--
```

Comment:
This seems oddly cut off and I had to read it a few times to get what the implied meaning was. It might just be best to say Rust doesn't have an aliasing model but let's talk about what aliasing is.